app/controllers/api/alert_definitions_controller.rb

```diff
@@ -23,5 +25,11 @@ def edit_resource(type, id = nil, data = {})
         raise BadRequestError, "Failed to update alert definition - #{err}"
       end
     end
+
+    private
+
+    def set_additional_attributes
+      @additional_attributes = %w(expression notify_email)
```

Comment:
why was notify_email needed ?

Reply:
Even though I'm curious about this as well, this gets us to green so I'm merging... Let's follow up.